crates/astria-telemetry/src/metrics/builders.rs

```diff
@@ -115,6 +113,16 @@ impl ConfigBuilder {
                 .map_err(|error| Error::StartListening(error.into()))?
         } else {
             let recorder = bucket_builder.builder.build_recorder();
+
```

Comment:
Above `fn build` (the scope of the lines I am commenting on): can you add a section that this spawns a task on the tokio runtime and will fail if running outside of tokio?

Reply:
Done in [780a46b](https://github.com/astriaorg/astria/pull/2221/commits/780a46bca9606a0af9a70250c2e825d4e9e7a86a).